build/automation/var/profile/local.mk

```diff
@@ -23,6 +23,7 @@ APP_ADMIN_PASSWORD = admin
 
 # TODO: What is this variable for? We always should be using https
 HTTP_PROTOCOL = http
+REQUEST_ID_HEADER =
```

Comment:
No, it shouldn't.  Remember this morning we said that the unique request ID would be generated by the proxy server. For this to work, we need to set this value to HTTP_X_REQUEST_ID when we are running in an environment that always goes through the proxy server, so that Django uses the request ID passed in rather than it generating a new one.  In our local environment, we can run the API as standalone across port 8000 and so in this case we do not run through a proxy server and so we need Django to generate a request ID for us.  I've set this variable to blank here in the case where we may have deployed to the dev env first, but now want to deploy locally, in which case we don't want this variable set. Does this make sense?